modules/processing/snowflake/sql/ST_DELAUNAYPOLYGONS.sql

```diff
@@ -0,0 +1,11 @@
+----------------------------
+-- Copyright (C) 2021 CARTO
+----------------------------
+
+CREATE OR REPLACE SECURE FUNCTION @@SF_PREFIX@@processing.ST_DELAUNAYPOLYGONS
+(points ARRAY)
+RETURNS ARRAY
+AS $$(
+    SELECT ARRAY_AGG(ST_ASGEOJSON(ST_MAKEPOLYGON(TO_GEOGRAPHY(unnested.VALUE))))
```

Comment:
At some point we can check if it's better here to cast the array elements to geojson objects or convert them to string just after. It shouldn't make any different I guess for converting them back using TO_GEOGRAPHY.

Reply:
Yep, casting is to be included